CON-4123: Availability block via channel manager api - documentation (#189)
MewsSystems/gitbook-channel-manager-api: mews-operations/operations.md

## Inventory

| <div style="width:200px">Operation</div>                                                                    | Description                                                                                                                |

> |:------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------|

Reviewer: revert the format

| [Confirm price update](inventory.md#confirm-price-update) | Confirms price update was processed succesfully or with errors. |
| [Confirm restriction update](inventory.md##confirm-reistriction-update) | Confirms restriction update was processed succesfully or with errors. |
| [Confirm availability block synchronization](availabilityBlock.md##confirm-availability-block-confirmation) | Confirms availability block synchronization was processed succesfully or with errors. |

> | [Process availability block](availabilityBlockProcessor.md#process-availability-block) | Process availability block that can be add, updat or cancel. |

Reviewer 2: ```suggestion
| [Process availability block](availabilityBlockProcessor.md#process-availability-block) | Add, update or cancel an availability block. |
```
